vendor/mozilla-rust-sdk/googleapis-raw/generate.sh

```diff
@@ -27,7 +41,8 @@ if ! [[ -x "$(command -v grpc_rust_plugin)" ]]; then
 fi
 
 echo "Pulling git submodules"
-git submodule update --init --recursive
+# comment out to work on master...
+#git submodule update --init --recursive
```

Comment:
What if we just uncomment this line so the submodule updates happen automagically? Is there a reason we don't want it to happen in the script?

Reply:
Yeah, I'd rather it be a mindful thing, partly because it could introduce a lot of breakage.